python/src/uagents/mailbox.py

```diff
@@ -164,12 +152,6 @@ async def register_in_agentverse(
             request.user_token, identity.address, agent_details, agentverse
         )
 
-    if request.agent_type == "mailbox" and not is_mailbox_agent(endpoints, agentverse):
```

Comment:
We no longer have the agent's endpoints, so it doesn't make sense to do this check here anymore.